tools/cloud_functions/gcs_event_based_ingest/tests/cli/test_backfill.py

```diff
@@ -0,0 +1,257 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""integrtion tests for gcs_ocn_bq_ingest"""
+import json
+import os
+import uuid
+from time import monotonic
+from typing import List
+
+import backfill
+import google.cloud.storage as storage
+import pytest
+from google.cloud import bigquery
+
+from gcs_ocn_bq_ingest import main
+
+TEST_DIR = os.path.realpath(os.path.dirname(__file__) + "/..")
+LOAD_JOB_POLLING_TIMEOUT = 20  # seconds
+
+
+@pytest.fixture(scope="module")
+def bq() -> bigquery.Client:
```

Comment:
these fixtures are all duplicated from `test_gcs_ocn_bq_ingest_it.py` should DRY them up into a common fixtures module.